pkg/server/handlers.go

```diff
@@ -62,14 +69,62 @@ func getEnvironment(w http.ResponseWriter, r *http.Request) {
 func patchEnvironment(w http.ResponseWriter, r *http.Request) {
 	pathVars := mux.Vars(r)
 
-	err := updateEnvironmentHosts(pathVars["org"], pathVars["env"], r.Header.Get("Authorization"))
+	ns, err := clientset.Core().Namespaces().Get(pathVars["org"] + "-" + pathVars["env"])
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
-		helper.LogError.Printf("Error syncing Hosts to the Environment: %v\n", err)
+		helper.LogError.Printf("Error getting existing Environment: %v\n", err)
 		return
 	}
 
-	w.WriteHeader(http.StatusNoContent)
+	getSecret, err := clientset.Core().Secrets(pathVars["org"] + "-" + pathVars["env"]).Get("routing")
```

Comment:
I believe the naming is an artifact of an older version of enrober where there may have been multiple k8s objects in a single handler. I'll rename it to `routingSecret` as at this time it contains both a public and private routing key as data.